### DISCUSSION TITLE
✨ Code view - Convert TODO comments to issues

### DISCUSSION THREAD

**Discussion by bradystroud** [community/community#58695] [Repositories]
### Select Topic Area

Product Feedback

### Body

GitHub should spot `TODO` comments in code, and offer an annotation to instantly turn these into trackable issues?

Example URLs

https://github.com/SSWConsulting/SSW.Website-v3/blob/9e0d1b58223010b213f52e853fff2bdc27fd87df/next-sitemap.config.js#L17

**Comment by Banaanae** [comment]
There is an action for this
https://github.com/marketplace/actions/todo-to-issue
